Package.swift

```diff
@@ -25,7 +25,7 @@ let package = Package(
         .package(name: "MapboxGeocoder", url: "https://github.com/mapbox/MapboxGeocoder.swift.git", from: "0.14.0"),
         .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", from: "0.10.8"),
         .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios.git", from: "45.0.2"),
-        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .exact("10.0.0-beta.15")),
+        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .revision("7425b7420fbf91d3666dff8dc229acdb44f2db33")),
```

Comment:
This is technically based on the latest commit in mapbox/mapbox-maps-ios#205, rather than main, so we’ll need to update it to a branch or tag before landing this PR, to avoid a reference to a commit hash that may go away due to a rebase upstream.

Reply:
Updated to `main` to get more recent changes.